bin/cassandra.in.sh

```diff
@@ -99,6 +99,7 @@ JAVA_OPTS="$JAVA_OPTS:-Djava.library.path=$CASSANDRA_HOME/lib/sigar-bin"
 CASSANDRA_TEMP_DIR=$SERVICE_HOME/var/data/tmp
 JVM_OPTS="${JVM_OPTS} -Djava.io.tmpdir=$CASSANDRA_TEMP_DIR"
 JVM_OPTS="${JVM_OPTS} -Djna.tmpdir=$CASSANDRA_TEMP_DIR"
+export LIBFFI_TMPDIR="$CASSANDRA_TEMP_DIR"
```

Comment:
We may also want to set and export `TMPDIR` for any other native libraries that do tricky things:

```suggestion
export TMPDIR="$CASSANDRA_TEMP_DIR"
```